packages/superset-ui-dimension/src/getTextDimension.ts

```diff
@@ -23,31 +15,17 @@ export default function getTextDimension(
   input: GetTextDimensionInput,
```

Comment:
It would be nice to add some documentation here about how to use getTextDimension. From what I can see, you have two options:

1. Don't provide an svg node, but provide the className and style details required to construct it.
2. Provide an svg node that you've already applied the className and style to; these parameters now have no effect

I don't think it's clear to the developer that providing an svg node and a className/style doesn't apply the className/style to the node. Either a nice docblock or splitting this into 2 different functions with 2 different inputs would make it more clear